### ISSUE TITLE
Painel Eletrônico controlado por IP dentro da mesma rede

### ISSUE THREAD

**Issue by CristianoCMLP** [interlegis/sapl#2120]
O painel eletrônico ele poderia se controlado por ip dentro na mesma rede, da forma que está ele necessita de dois monitores um para o controlador e outro para exibir o resultado.  Isto poderia diminuir os custos da implantação do painel eletrônio, pois uma smatv poderia ser colocada em qualquer plenário.

**Comment by edwardoliveira** [comment]
Sim, excelente ideia (de verdade). Poderemos implementar algo assim no decorrer do ano de 2019, **provavelmente**, quando iremos reprojetar essa parte.

**Comment by rosenocmcc** [comment]
Eu uso o painel em duas tvs simultânea conectado por um chromecast. Poderiamos ter uma tela exclusiva com o cronometro para quando o parlamentar utilizar a tribuna. No nosso hoje fica muito pequeno os cronometros.
